indico/modules/events/templates/display/indico/_common.html

```diff
@@ -89,6 +89,7 @@
 
 {% macro render_datetime(prop_name, dt, timezone, format='medium') -%}
     <time itemprop="{{ prop_name }}" datetime="{{ dt.isoformat() }}">
+        {{ dt.strftime('%A') }}
```

Comment:
![dates](https://cloud.githubusercontent.com/assets/9071624/19391332/69ec05a8-922c-11e6-9516-97443bdcded2.png)
Im not sure what you mean, i get this when using format='full' , its good but that timezone name is really long